test/Target/llvmir.mlir

```diff
@@ -802,7 +802,7 @@ func @cond_br_arguments(%arg0: !llvm.i1, %arg1: !llvm.i1) {
   llvm.br ^bb1(%arg1 : !llvm.i1)
 }
 
-// CHECK-LABEL: define void @llvm_noalias(float* noalias) {
+// CHECK-LABEL: define void @llvm_noalias(float* noalias {{%*.}}) {
```

Comment:
You can simplify all these checks by being simply like: `CHECK-LABEL: define void @llvm_noalias(`

Reply:
does it mean that signature of the functions is not need to be checked? I mean like the number of the arguments and their types?

Reply:
Mmmm, actually I was wrong here, here we may be checking the type of the argument.

In general tests are easy to be written "over constrained"

Reply:
@joker-eph got it, thanks!

Reply:
Indeed, function argument types are also checked here (but should probably be separated in other tests)